src/target/riscv/riscv-013.c

```diff
@@ -3415,43 +3415,61 @@ static void log_mem_access_result(struct target *target, bool success, riscv_mem
 
 typedef enum {
 	MEM_ACCESS_RESULT_TYPE_OK,
-	MEM_ACCESS_RESULT_TYPE_DISABLED,
+	MEM_ACCESS_RESULT_TYPE_COMMON,
```

Comment:
Instead of `COMMON`, it looks like you can directly say `DISABLED`. That way it should be more clear to the reader.

```suggestion
	MEM_ACCESS_RESULT_TYPE_DISABLED,
```

Reply:
Addressed